doc/nrf/protocols/wifi/promiscuous_operation.rst

```diff
@@ -0,0 +1,155 @@
+.. _ug_nrf70_developing_promiscuous_packet_reception:
+
+Raw IEEE 802.11 promiscuous packet reception
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series devices support Promiscuous mode.
+Promiscuous mode of operation is only supported in connected Station mode.
+In Promiscuous mode of operation, all received IEEE 802.11 packets that belong to the same BSS are sent to the receiving application by the nRF Wi-Fi driver.
```

Comment:
```suggestion
In Promiscuous mode of operation, all received IEEE 802.11 packets that belong to the connected BSS are sent to the receiving application by the nRF Wi-Fi driver.
```

Reply:
I think the correct name is "nRF Wi-Fi driver", not with the registered mark. The registered mark is used with the first instance of Wi-Fi in the document.

Reply:
Oh, is it different than: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/2.4.99-dev2/nrf/drivers/wifi/nrf700x/nrf700x.html ?
In this doc, "nRF70 Series Wi-Fi driver" is used throughout

Reply:
"nRF70 Series Wi-Fi driver" is correct, typo in my last comment.